sycl/unittests/pi/level-zero/spirv-source.hpp

```diff
@@ -0,0 +1,61 @@
+//==---- spirv-source.hpp --- PI unit tests
+//---------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// Generated from the following OpenCL kernel code:
```

Comment:
Is there a command-line for generating the below spirv you could stick in this comment?

Reply:
Done!